### PR TITLE
test(protocol-designer): fix e2e tests after vite migration

### PR DESCRIPTION
**Pull request by shlokamin**
# Overview

This PR fixes Protocol Designer e2e tests from the vite migration. It also removes references to "beta" within PD cuz... it is not in beta.

Closes [AUTH-86](https://opentrons.atlassian.net/jira/software/c/projects/AUTH/issues/AUTH-86)

# Changelog

- Fix Protocol Designer e2e tests

# Review requests

Make sure PD e2e tests are passing

# Risk assessment

Low

[AUTH-86]: https://opentrons.atlassian.net/browse/AUTH-86?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: step-generation/src/commandCreators/atomic/engageMagnet.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ export const engageMagnet: CommandCreator<EngageMagnetArgs> = (
   const { module: moduleId, engageHeight } = args
   const commandType = 'magneticModule/engage'
 
-  if (module === null) {
+  if (moduleId === null) {
```
Comment (shlokamin) [inline step-generation/src/commandCreators/atomic/engageMagnet.ts:16]: this was a bug that slipped through years ago... `module` is a special global keyword that webpack injected so we never noticed the bug. vite does not inject it, so we got a null access error

File: protocol-designer/README.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
-# Opentrons Protocol Designer Beta
```
Comment (jerader) [inline protocol-designer/README.md:1]: lol thanks for removing this

File: protocol-designer/README.md [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
+# Opentrons Protocol Designer
 
 ## Overview
 
 Protocol Designer is a tool for scientists and technicians to create protocols for their [OT-2 personal pipetting robot][ot-2] without having to write any code. It provides visual feedback including liquid tracking and tip tracking to allow users to see exactly what their protocol will do at each step. The protocols are saved to Opentrons JSON Protocol files, which can be uploaded to the Opentrons Desktop App to run on a robot.
 
-Protocol Designer Beta is optimized for [Chrome][chrome] browser. Other browsers are not fully supported.
```
Comment (jerader) [inline protocol-designer/README.md:7]: and here too

**Review by jerader — APPROVED** [review]
thank you!!

**Pull request merged**